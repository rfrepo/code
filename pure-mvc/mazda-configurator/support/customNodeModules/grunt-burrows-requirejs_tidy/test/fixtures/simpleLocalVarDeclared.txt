define([
    'support/class1',
    'class2'
], function () {
    'use strict';
    puremvc.define({

        bmc.support.class1.CONFIGURABLE_ITEM_UI;

        function thisIsAFunction() {
            var class1 = bmc.support.class1;
        };

        function anotherFunction() {
            var otherVar,
                class1 = bmc.support.class1;
         };

         function anotherFunction() {
            var otherVar,
                class1 = bmc.support.class1,
                secondVar;
         };

        function finalFunction() {
            var class1 = bmc.support.class1,
                finalVar;
        };
    });
});