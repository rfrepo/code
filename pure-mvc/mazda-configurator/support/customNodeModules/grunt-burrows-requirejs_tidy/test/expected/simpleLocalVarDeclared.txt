define([
    'support/class1',
    'class2'
], function () {
    'use strict';
    var class1 = arguments[0];

    puremvc.define({

        class1.CONFIGURABLE_ITEM_UI;

        function thisIsAFunction() {
            
        };

        function anotherFunction() {
            var otherVar;
         };

         function anotherFunction() {
            var otherVar,
                secondVar;
         };

        function finalFunction() {
            var finalVar;
        };
    });
});